<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateDialogBase</class>
 <widget class="QDialog" name="DateDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Select date</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Date to edit:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okbutton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <number>8388687</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelbutton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <number>8388675</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCalendarWidget" name="datePicker"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="todayButton">
       <property name="text">
        <string>today</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/cr16_action_bookmark</normaloff>:/cr16_action_bookmark</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="weekSelector"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>datePicker</tabstop>
  <tabstop>todayButton</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>weekSelector</tabstop>
  <tabstop>okbutton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
